CHANGELOG.md

```diff
@@ -13,11 +13,21 @@ Our versioning strategy is as follows:
 
 ### 🛠 Breaking Changes
 
-* [sitecore-jss] Switch to edge site query for XP and gets config sites + sxa sites (ignoring website)
+* `[sitecore-jss]` Switch to edge site query for XP and gets config sites + sxa sites (ignoring website)
   * Previously introduced Boolean `useSiteQuery` switch for XMCloud users has been removed.
   * Search query usage has been removed.
   * If you have any non-nextjs sites they should filter them out in multisite config plugin
 
+* `[sitecore-jss-nextjs]` Deprecated exports have been removed ([#1780](https://github.com/Sitecore/jss/pull/1780)):
```

Comment:
Should we just group all the changes below together and provide a link to this PR?
e.g. * Removed deprecated functionality (...): 1, 2, 3 ...

Reply:
I don't think it makes much of a difference. It will be either
- Deprecations: 1, 2, 3
or just
- 1, 2 ,3 (the way it is now)